docs/end_to_end_acknowledgements.md

```diff
@@ -0,0 +1,63 @@
+
+## End To End Acknowledgements
```

Comment:
Using "end to end" here is an adjective, so I believe you should use hyphens.

"End-to-end acknowledgements"

Please use that throughout.

Also, the OpenSearch documentation page is capitalizing only the first letter in a header for common nouns. So, this header should be:

```
## End-to-end acknowledgements
```